react-components/src/architecture/concrete/terrain/geometry/RegularGrid2.ts

```diff
@@ -18,7 +18,7 @@ export class RegularGrid2 extends Grid2 {
   public readonly origin: Vector2;
   public readonly increment: Vector2;
 
-  private _buffer: Float32Array; // NaN value in this array means undefined node
+  private _buffer: Float32Array<ArrayBuffer>; // NaN value in this array means undefined node
```

Comment:
This broke in type checking for me locally. I'm suspecting that the type checker in CI doesn't catch this properly, but I include it anyway. It's an issue we should definitely check promptly